### ISSUE TITLE
Date issues with Raspberry Pi 4

### ISSUE THREAD

**Issue by sasimpson** [debuerreotype/docker-debian-artifacts#150]
Had an issue with a docker image which is based on debian:11.2, originally based on debian:10.  example:

```
$ docker run -it --rm debian:buster date 
Wed Jan 26 02:30:14 UTC 2022
```
vs
```
docker run -it --rm debian:bullseye date 
Thu Jan  1 00:00:00 UTC 1970
```

on the host os, Raspian, i was able to fix by adding:
```
deb http://raspbian.raspberrypi.org/raspbian/ testing main
```

and installing:
```
apt-get install libseccomp2/testing
```

happy to contribute any other information.  this seems to be weird issue that is hard to track down.  not sure if its even an issue with this particular repository.  if not, please direct, thanks!

 https://github.com/dotnet/dotnet-docker/issues/3253

**Comment by tianon** [comment]
TLDR, you'll need to update your host, specifically to:

- Docker version 20.10.8 or newer
- libseccomp version 2.4.2 or newer

See moby/moby#40734 for more details around this (and similar issues).

**Comment by sasimpson** [comment]
thank you!
